docs/shared/migration/from-turborepo.md

```diff
@@ -145,7 +145,7 @@ For each `turbo.json` configuration property, the equivalent Nx property is list
 | `--parallel`                | N/A                                                                                                                                                                                   |
 | `--remote-only`             | N/A. Can [ignore the remote cache](/ci/features/remote-cache#skipping-cloud-cache) with `--no-cloud`.                                                                                 |
 | `--summarize`               | N/A                                                                                                                                                                                   |
-| `--token`                   | Set the [Nx Cloud token in `nx.json`](/ci/recipes/security/access-tokens#setting-access-tokens) or as an environment variable (`NX_CLOUD_ACCESS_TOKEN`)                               |
+| `--token`                   | Set the [Nx Cloud token in `nx.json`](/ci/recipes/security/access-tokens#setting-ci-access-tokens) or as an environment variable (`NX_CLOUD_ACCESS_TOKEN`)                            |
```

Comment:
Is the equivalent here the `CI Access Token` or are people going to get confused and try to pass in their NxCloudId